cloudpickle/cloudpickle.py

```diff
@@ -1218,6 +1229,19 @@ class id will also reuse this enum definition.
     return _lookup_class_or_track(class_tracker_id, enum_class)
 
 
+def _get_parent(module):
```

Comment:
To make code easier to follow, I think this should better be named `_get_parent_module_or_package`.